nixos/roles/github-actions-runner.nix

```
@@ -1,11 +1,16 @@
 { lib, config, pkgs, ... }:
 let
   cfg = config.roles.github-actions-runner;
+  queued-build-hook = builtins.fetchTarball {
```

`builtins.fetchurl` is harder to cache i.e. binary cache. It's better to use a proper derivation:

```suggestion
  queued-build-hook = pkgs.fetchurl {
```